api/pkg/client/mlflow/mlflow_test.go

```diff
@@ -205,28 +209,76 @@ var RunFailedDeleteRun = `
 	}
 }`
 
-var DeleteExperimentDoesntExist = `
-{
-    "error_code": "RESOURCE_DOES_NOT_EXIST",
-    "message": "No Experiment with id=999 exists"
-}`
-
 var RunDoesntExist = `
 {
     "error_code": "RESOURCE_DOES_NOT_EXIST",
-    "message": "Run with id=unknownId not found"
+    "message": "run with id=unknownId not found"
 }`
 
 var DeleteRunAlreadyDeleted = `
 {
     "error_code": "INVALID_PARAMETER_VALUE",
-    "message": "The run xytspow3412oi must be in the 'active' state. Current state is deleted"
+    "message": "the run xytspow3412oi must be in the 'active' state. Current state is deleted"
 }`
 
-var FailedDeleteArtifact = `
-{
-    "message": "The run xytspow3412oi must be in the 'active' state. Current state is deleted"
-}`
+func TestNewMlflowService(t *testing.T) {
+	httpClient := http.Client{}
+	gcsConfig := GcsArtifactConfig{Ctx: context.Background()}
+	api, _ := storage.NewClient(gcsConfig.Ctx)
+
+	tests := []struct {
+		name           string
+		artifactType   string
+		gcsConfig      *GcsArtifactConfig
+		expectedError  error
+		expectedResult *mlflowService
+	}{
+		{
+			name:          "Mlflow Service with GCS Artifact",
+			artifactType:  "gcs",
+			gcsConfig:     &gcsConfig,
+			expectedError: nil,
+			expectedResult: &mlflowService{
+				API:             &httpClient,
+				Config:          Config{TrackingURL: "", ArtifactServiceType: "gcs", ArtifactServiceGcsConfig: &gcsConfig},
+				ArtifactService: artifact.NewGcsArtifactService(api),
```

Comment:
Tiny nit but if you don't want this test to appear as if it's testing `NewGcsArtifactService` and `NewNopArtifactService`, you can probably just initialise the struct directly instead of calling those methods.